src/main/webapp/mentors.js

```diff
@@ -1,18 +1,22 @@
 /**
  * Get mentors from the mentor servlet.
+ * @param {String} response
  */
 function getMentors() {
   console.log('entering get mentors function/n');
   let url = new URL('/mentors', location.protocol + '//' + location.hostname);
 
   fetch(url).then(errorHandling).then((response) => response.json())
       .then((mentors) => {
-    console.log(mentors);
     const mentorsContainer = document.getElementById('mentors-container');
     mentorsContainer.innerHTML = '';
     for (const mentor of mentors) {
-      mentorsContainer
-          .appendChild(createMentorElement(mentor.name, mentor.description));
+      for (const propName in mentor) {
+        const propVal = mentor[propName];
+        console.log(propName, propVal);
+      }
```

Comment:
It is slightly different in that it reports the properties explicitly (instead of giving an object which you can then inspect). However, if you think it is simpler/better, I can switch to console.log(mentor) and not lose any information.

Reply:
Okay, thanks for the clarification! Totally up to you if you want to keep it or not.